content/docs/introduction/faq.mdx

```diff
@@ -51,11 +51,23 @@ are not functional in a self-hosted Tina setup.
 
 The self-hosted backend does not currently have endpoints to support search functionality.
 
-## How do you enable co-author on Git commits?
+## Enabling co-author on Git commits
 
-By default when using TinaCloud, the author on Git commits is the TinaCloud GitHub [app](https://github.com/apps/tina-cloud-app). It is possible to [co-author](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) Git commits in order to attribute the commit to an additional user. This can be enabled per-user in the TinaCloud account [settings](https://app.tina.io/account/settings). By default, the name and email of the co-author is configured according to your TinaCloud account, but they can be set to any valid name and email address. Once configured, click the "Enable co-authoring" toggle and the "Save Settings" button.
+By default when using Tina Cloud, the author on Git commits is the Tina Cloud GitHub [app](https://github.com/apps/tina-cloud-app).  Some users prefer to know who made the change.
+
+For this use case, it is possible to [co-author](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) Git commits in order to attribute the commit to an additional user. This can be enabled per-user in the Tina Cloud account [settings](https://app.tina.io/account/settings). 
```

Comment:
@bradystroud

- [ ] Because you have the same link to "settings" in line 64, I wouldn't repeat it here 

as per https://www.ssw.com.au/rules/avoid-redundant-links/